cmapi/integration_tests/integration_tests.yml

```diff
@@ -0,0 +1,23 @@
+# Ansible provisioning for integration tests
+
+- hosts: "primary,replicas"
+  become: yes
+  become_user: root
+  gather_facts: no
+
+  tasks:
+    # Install cs_package_manager
+    - name: "Download cs_package_manager"
+      get_url:
+        url: "https://raw.githubusercontent.com/mariadb-corporation/mariadb-columnstore-engine/refs/heads/cs_package_manager_v3.5/cmapi/scripts/cs_package_manager.sh"
```

Comment:
We do have this script in columnstore repo.
As I know the newest version and all changes are there. Moreover we distribute it with CMAPI package.